dwm.c

```diff
@@ -4319,7 +4319,14 @@ updatebarpos(Monitor *m)
 	m->ww = m->mw;
 	m->wh = m->mh;
 	Bar *bar;
-	#if BAR_PADDING_PATCH
+	#if BAR_GAPS_PADDING_PATCH
+	int y_pad = 0;
+	int x_pad = 0;
+	if (enablegaps) {
+		y_pad = gappov;
+		x_pad = gappoh;
+	}
```

Comment:
Perhaps this could be simplified to:

```c
	int y_pad = (enablegaps ? gappoh : 0);
	int x_pad = (enablegaps ? gappov : 0);
```

Reply:
I intentionally wrote it like that to avoid repetition. It also seems logical to not to evaluate the condition twice.